web/src/pages/Settings/Config/ConfigPanels/LMSMode.jsx

```diff
@@ -20,12 +21,22 @@ export default function LMSMode() {
         setLoading(false);
     };
 
+    function LMSControlLink() {
+        if(lmsMode){
+            return (
+                <Button variant="contained" sx={{color: 'black'}} target="_blank" href={`http://${window.location.hostname}:9000`}>
+                    LMS Control Panel
+                </Button>
+            )
+        }
+    }
+
     function LMSModal(){
         if(lmsMode){
             return(
                 <ConfigModal
                     body={"This will reset AmpliPi to factory settings, you will have to either manually reconfigure it or reupload the config that was downloaded when LMS mode was initially toggled."}
-                    onApply={() => {ConfigDownload(); LMSModeHandler();}}
+                    onApply={() => {LMSModeHandler();}}
                     open={modalOpen}
```

Comment:
not a fan of this change: https://github.com/micro-nova/AmpliPi/issues/976#issuecomment-2411833840 & https://github.com/micro-nova/AmpliPi/pull/805